lavamoat/build-system/policy.json

```diff
@@ -6222,6 +6194,11 @@
         "eslint-plugin-prettier>prettier-linter-helpers>fast-diff": true
       }
     },
+    "process": {
```

Comment:
Looks like the previous version was not detecting it. Harmless, but worth checking out why.